app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
~A partial should only contain a content part of a page. This belongs in your layout~ 

Skip that: Remove this file